core/src/tangram.h

```diff
@@ -11,12 +11,27 @@ namespace Tangram {
 
 class DataSource;
 
+enum LabelType {
+    icon,
+    text,
+};
+
 struct TouchItem {
     std::shared_ptr<Properties> properties;
     float position[2];
     float distance;
 };
 
+using FeatureSelectionCallback = std::function<void(const std::vector<TouchItem>&)>;
+
+struct TouchLabel {
```

Comment:
`LabelPickResult` seems more explicit, along with `LabelPickCallback` and al.
We can also reflect that in the android interface.